hera/editor/lib/WinConditionCard.tsx

```diff
@@ -192,6 +192,37 @@ export default function WinConditionCard({
               </Stack>
             </label>
           )}
+          {condition.type !== WinCriteria.Default && (
+            <>
+              <label>
+                <Stack gap start>
+                  <span className="input-label">
+                    <fbt desc="Label for optional win condition checkbox">
+                      Optional
+                    </fbt>
+                  </span>
+                  <input
+                    checked={condition.optional}
+                    onChange={(event) =>
+                      onChange({
+                        ...condition,
+                        optional: event.target.checked,
+                      })
+                    }
+                    type="checkbox"
+                  />
+                </Stack>
+              </label>
+              {condition.optional && (
+                <p>
+                  <fbt desc="Description for optional win conditions">
+                    * Optional win conditions do not end the game when
+                    fulfilled.
+                  </fbt>
```

Comment:
Let's remove the star, and do: Optional conditions do not end the game when fulfilled.

Reply:
Also, let's move this whole section to below the Secret section.

Reply:
It's done in 63c50f86803d14538d10d1d94c63eedb48808354